js/src/elements/form-behavior.ts

```diff
@@ -5,8 +5,13 @@ export const defaultMessageStrings = {
 	inputMaxLength: '{inputLabel} cannot be longer than {maxLength} characters.',
 	inputMinLength: '{inputLabel} must be at least {minLength} characters.',
 	inputRequired: '{inputLabel} is required.',
+	notAuthenticated:
+		'You are not authenticated, please refresh the page and try again. If this issue persists please log out and log back in.',
+	notAuthorized:
+		'You are not authorized to make this response, if you believe this to be in error please refresh the page and try again.',
 	pleaseFixTheFollowingIssues: 'Please fix the following issues to continue:',
 	thereAreNoEditsToSubmit: 'There are no edits to submit.',
+	tooManyRequests: 'You have sent too many requests, please wait a few minutes and try again.',
```

Comment:
This should also be split:

```suggestion
	tooManyRequests: 'You have sent too many requests. Please wait a few minutes and try again.',
```